### PR TITLE
Multiplayer debug menu adjustments, Modify Tile permissions

### PR DESCRIPTION
**Pull request by wolfreak99**
I was only able to test it on a localhost form, as I am on a public wifi connection right now with no port forward access, but it seems to work fine enough.

### CODE REVIEW

**Comment by Broxzier** [comment]
Console should probably also be greyed out, and I'd like to not update the version number yet until everything has been tested, to prevent conflicts during a rebase later on. There's a tag on the PR that will not be removed before the version has been updated.

File: data/language/en-GB.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# STR_XXXX part is read and XXXX becomes the string id number.
+﻿# STR_XXXX part is read and XXXX becomes the string id number.
```
Comment (Broxzier) [inline data/language/en-GB.txt]: I think you accidentally added a byte order mark here.

File: src/openrct2/network/NetworkAction.cpp [hunk 2]
```diff
@@ -221,7 +221,12 @@ const std::vector<NetworkAction> NetworkActions::Actions =
         {
             MISC_COMMAND_PASSWORDLESS_LOGIN
         }
-    },
+	}, {
+		STR_ACTION_MODIFY_TILE, "PERMISSION_MODIFY_TILE",
+		{
+			GAME_COMMAND_MODIFY_TILE
+		}
+	}
```
Comment (Broxzier) [inline src/openrct2/network/NetworkAction.cpp]: Use spaces.

File: src/openrct2/network/network.cpp [hunk 3]
```diff
@@ -784,6 +785,9 @@ void Network::LoadGroups()
 		}
 		json_decref(json);
 	}
+
+	// Host group should always contain all permissions.
+	group_list.at(0)->ActionsAllowed.fill(0xFF);
```
Comment (Broxzier) [inline src/openrct2/network/network.cpp]: What does the magic number here mean?

Comment (wolfreak99) [inline src/openrct2/network/network.cpp]: What magic number? the 0 or the 0xFF?

**Comment by wolfreak99** [comment]
@Broxzier I chose to keep console active because it's still accessible via the tilde key, and it would be kinda weird to have console accessible by the hotkey but not by the menu, and there's also a few console functions that are multiplayer compatible. Let me know if you still think it should be disabled

File: src/openrct2/network/network.cpp (hunk 3 above)
Comment (Broxzier) [inline src/openrct2/network/network.cpp]: The 0 is clear, this returns the first in the group which is the server I assume, ad `0xFF` sets all flags or something?

**Comment by Nubbie** [comment]
In my opinion which no one asked for;
I would like to see the console in the list **and** the options being grayed out use a tooltip saying 'Not available in multiplayer' as they probably will be enabled in the future

**Comment by wolfreak99** [comment]
Dropdown menus do not yet have the ability to have tooltips.

**Comment by wolfreak99** [comment]
This can be deleted now, yes?

**Comment by Broxzier** [comment]
I cleaned up the commits and merged them now in a separate PR. I wanted to change these, but that didn't work as I had hoped.

**Pull request closed without merging**